docs/platforms/java/migration/7.x-to-8.0.mdx

```diff
@@ -6,6 +6,8 @@ description: "Learn about migrating from version 7.x to 8.0.0"
 
 ## Migrating from `io.sentry:sentry` `7.x` to `io.sentry:sentry` `8.0.0`
 
+Please make sure to use the same version for all Sentry dependencies.
```

Comment:
```suggestion
Please make sure to use the same version for all Sentry dependencies, otherwise you might face an `AbstractMethodError` runtime error.
```

for better SEO